docs/development-guide.md

```diff
@@ -62,7 +62,7 @@ coverage rather than decreasing it.
 
 We use [pytest][pytest-docs] as our testing framework.
 
-#### Stages
+### Stages
```

Comment:
Something seemed odd with the level of the headings in this document (on this line and [this one](https://github.com/wayfair-incubator/columbo/pull/115/files#diff-da18896de895660f32863b2a6d21a7b1e7c1095e4b100f48764ddc3ac9157772R88)). I've made a best-guess at what they should be.